### PR TITLE
week1 homework

### PR DESCRIPTION
**Pull request by hemagunti**
I have completed python week1 homework

### CODE REVIEW

File: Week1/main.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
+#1.print my name and age
+name = "Hema"
+age = 37
+print(f"Name:{}\n Age:{}".format(name, age))
```
Comment (Rassibassi) [inline Week1/main.py]: hej! You are mixing f-strings, with the `format` method!

its either with `format`:
```python
"Name:{}\n Age:{}".format(name, age)
```
or with f-strings:
```python
f"Name:{name}\n Age:{age}"
```

checkout here: https://realpython.com/python-f-strings/ to read up on this topic

File: Week1/main.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+#4. A Python script to add a key to a dictionary
+sample_dictionary= {0: 10, 1: 20}
+sample_dictionary[2] = 30
+print(sample_dictionary)
+
+#5. Python program to calculate body mass index.
+height = float(input("Enter your body weight in kg"))
+weight = float(input("Enter your body height in feet"))
+print("Your body mass index is: ", round(weight / (height * height)))
```
Comment (Rassibassi) [inline Week1/main.py]: Python has a method to write `height * height` more elegantly, try to use the power to operator

File: Week1/main.py [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+#8. Create a list of 5 city names and convert it into tuples.
+my_list=["Copenhagen","Herlev","Roskilde","Malmo","Gentofte"]
+print(my_list)
+my_tuple=tuple(my_list)
+print(my_tuple)
+
+#9. Remove duplicated from the list:
+sample_list = [10,20,30,20,10,50,60,40,80,50,40]
+original_list = list(dict.fromkeys(sample_list))
```
Comment (Rassibassi) [inline Week1/main.py]: although `dict.fromkeys` does the job. It's not a method to be used here as it is not intended for making a unique list of things. checkout `set` or write a for loop

File: Week1/main.py [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+print(original_list)
+
+#10.remove the characters which have odd index values of a given string
+original_string = input("Please Enter your Own String : ")
+final_string = ''
+for i in range(len(original_string)):
```
Comment (Rassibassi) [inline Week1/main.py]: a more pythonic way to write your for loop is this:
```python
for i,character in enumerate(original_string):
    print(i, character)
```
Can you rewrite the for loop with this improvement?

File: Week1/main.py [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+     if (i % 2 == 0):
+          final_string = final_string + original_string[i]
```
Comment (Rassibassi) [inline Week1/main.py]:
```python
a = a + b
```
is the same as
```python
a += b
```

**Review by Rassibassi — COMMENTED** [review]
Good job see my inline comments!

File: Week1/main.py (hunk 4 above)
Comment (hemagunti) [reply Week1/main.py]: Thanks for letting me know about enumerate

**Pull request merged**